src/libs/cropOrRotateImage/utils.ts

```diff
@@ -0,0 +1,16 @@
+import {SaveFormat} from 'expo-image-manipulator';
+
+function getSaveFormat(type: string) {
+    switch (type) {
+        case 'image/png':
```

Comment:
should this be moved to CONST? instead of switch case?

Reply:
I moved the file type to const, but I think it looks better with switch/case than if we just create an object that maps gif and bmp to jpeg.